test/spec/geo/shared-tests-for-cartodb-layer-group-views.js

```diff
@@ -1,389 +0,0 @@
-var _ = require('underscore');
```

Comment:
Why are we removing this and duplicating the tests instead?

Reply:
This was/is kind of convenient IMO.

Reply:
Jasmine logs/workflow are easier to read this way (imo)